spec/unit_spec.rb

```diff
@@ -4,16 +4,16 @@
 expects = {
   10 => 10,
   10.0 => 10.0,
-  '10y' => 315360000,
-  '10w' => 6048000,
-  '10d' => 864000,
-  '10h' => 36000,
+  '10y' => 315_360_000,
+  '10w' => 6_048_000,
+  '10d' => 864_000,
+  '10h' => 36_000,
   '10m' => 600,
   '10s' => 10,
   '10x' => 10,
   '10' => 10,
   '10.0' => 10,
-  '10.0x' => 10,
+  '10.0x' => 10
 }
 
 describe 'to_seconds' do
```

Comment:
thanks, this looks good. Since the fix is in the middle of a PR (usually there's no need to keep two commits if it is a fix during code review), can you fold this into first commit (git rebase) then do a force push of this PR?